### PR TITLE
[nextbike] Networks update

### PR DESCRIPTION
**Pull request by adamdanielczyk**
Some more nextbike updates

### CODE REVIEW

**Comment by adamdanielczyk** [comment]
@eskerda I've added today another commit with more nextbike updates

**Comment by eskerda** [comment]
@adamdanielczyk `santander-monchengladbach-monchengladbach` seems to fail, can you check?

**Comment by adamdanielczyk** [comment]
@eskerda I've removed it for now - https://nextbike.net/maps/nextbike-live.xml?domains=sn&get_biketypes=1 doesn't seem to return anything for such domain, even though this json contains the city https://api.nextbike.net/maps/nextbike-live.json?list_cities=1

File: pybikes/data/nextbike.json [hunk 1]
```diff
@@ -2494,10 +2577,11 @@
             }
         },
         {
-            "domain": "np",
-            "tag": "nextbike-jastrzebie-zdroj",
+            "domain": "pj",
+            "tag": "system-rowerow-miejskich-jastrzebie-zdroj-jastrzebie-zdroj",
```
[the remark this answers is not in the document]
Comment (adamdanielczyk) [reply pybikes/data/nextbike.json]: No reason, this is done automatically by the script I'm using - I'll revert the change then

**Comment by eskerda** [comment]
Looking good, thanks @adamdanielczyk!

**Pull request merged**